indico/modules/rb/client/js/components/App.module.scss

```diff
@@ -103,3 +103,7 @@
     min-height: calc(100vh - #{$menu-height} - var(--extra-bars, 0) * 50px - 50px);
   }
 }
+
+:global(.ui.dimmer) {
+  backdrop-filter: blur(0.5rem);
+}
```

Comment:
Please don't include this here, it's basically overriding SUI behavior and "re-implementing" a feature SUI already provides  if we want this behavior, the `<Modal>` components should set `dimmer="blurring"`.